src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Protocol.cs

```diff
@@ -13,14 +13,14 @@
 
 namespace System.Net.Security
 {
-    // SecureChannel - a wrapper on SSPI based functionality.
-    // Provides an additional abstraction layer over SSPI for SslStream.
-    internal sealed class SecureChannel
+    internal delegate X509Certificate2? SelectClientCertificate(out bool sessionRestartAttempt);
```

Comment:
This seems duplicate of `SelectClientCertificateCallback` below

Reply:
fixed